pkg/kyverno/test/test_command.go

```diff
@@ -155,6 +153,69 @@ func Command() *cobra.Command {
 				}
 			}()
 
+			mStatus, _ := cmd.Flags().GetBool("manifest-mutate")
+			vStatus, _ := cmd.Flags().GetBool("manifest-validate")
```

Comment:
This is `vStatus` yet the flag shown is `-l`. Why?

Reply:
`v` flag is already so I had to come up with another one. WDYT?

Reply:
Any recommendation here?

Does the output for `-h` and `-m` make sense? Thanks. 🔹

Reply:
Maybe don't even use short flags but only long flags. The `-h` output makes sense, yes. The `-m` flag also looks good.

Reply:
Sounds good, thanks. I'll wrap up those soon. 🗒️